packages/ec-core/src/distributions/one_of_macro.rs

```diff
@@ -73,7 +73,14 @@ macro_rules! uniform_distribution_of {
 }
 
 #[cfg(test)]
-#[allow(clippy::unwrap_used)]
+#[rustversion::attr(before(1.81), allow(clippy::unwrap_used))]
+#[rustversion::attr(
+    since(1.81),
+    expect(
+        clippy::unwrap_used,
+        reason = "Unwrap is the best way to deal with errors in unit tests"
```

Comment:
I think unwrap is actually the best way in tests, since it actually gives file names and locations for _where_ in the test it failed - which error propagation doesn't. I can change it to "panicking" tho if you like that language more because it includes `.expect` and so on.

Reply:
I think I do like "panicking" better than just sticking with `.unwrap()`. I probably ought to use `.expect()` more often anyway, and this is at least consistent with that.